web/locales/es.json

```diff
@@ -104,28 +104,28 @@
         "legend": "Leyenda",
         "faq": "Preguntas frecuentes"
     },
-    "wind": "eólica",
-    "solar": "solar",
-    "hydro": "hidráulica",
+    "wind": "energía eólica",
+    "solar": "energía solar",
+    "hydro": "energía hidráulica",
     "hydro storage": "almac. hidrául.",
     "battery storage": "almac. en baterías",
     "biomass": "biomasa",
     "nuclear": "nuclear",
-    "geothermal": "geotérmica",
+    "geothermal": "geotermia",
```

Comment:
Would you mind explaining this one? I've seen both terms used a lot, but haven't been able to tell the difference :)

Reply:
Yeah, they're pretty much synonyms. I would say that "geotermia" is used more often in the context of geothermal heat pumps, but it also applies to geothermal energy for electricity production. I made the change here so that I didn't have to add "energy" to "geotérmica", because "X% of electricity is produced by geotermia" sounds good on its own. Hope I explained myself :P

Reply:
Thanks for the explanation, I'm happy to go with your suggestion in that case :)